### DISCUSSION TITLE
Generating the conditional prboability

### DISCUSSION THREAD

**Discussion by kenneth-lee-ch** [agrumery/aGrUM#10] [Q&A] [answered]
Is there a way in the PyAgrum package where it can learn the probability tables from the finite discrete sample by computing the frequency of the sample?

**Comment by phwuil** [comment, marked as answer]
in an URL : https://pyagrum.readthedocs.io/en/1.7.1/BNLearning.html#pyAgrum.BNLearner.fitParameters
in a code :
```python
learner=gum.BNLearner("test.csv")
learner.fitParameters(bn)
```

in a notebook :
![image](https://user-images.githubusercontent.com/5980636/230286922-9ad6bc3d-545a-40df-95be-009bc6a5ad27.png)

**Comment by phwuil** [comment]
Feel free to close the discussion when you think it is answered :-)
